src/Shellbar/Shellbar.js

```diff
@@ -121,6 +122,13 @@ class Shellbar extends Component {
         return (
             <div className={shellbarClasses}>
                 <div className='fd-shellbar__group fd-shellbar__group--product'>
+                    {backAction && <Icon
+                        className='fd-shellbar__group fd-button--back'
+                        glyph='nav-back'
+                        onClick={backAction}
+                        size='l'
+                        style={{ color: 'white', marginRight: '0.5em', flex: 'none' }} />
```

Comment:
We don't like to include inline styles - is there something missing in fundamental-styles? If so, we need to make a pull request there. If the style is available in fundamental-styles, but fundamental-react isn't on the correct version, we can update.